/**:
  ros__parameters:
    enable_object_collision_estimator: true
    goal_distance_thresh: 1.0
    heading_weight: 0.1
    stop_velocity_thresh: 0.1
    subroute_goal_offset_lane2parking: 7.5
    subroute_goal_offset_parking2lane: 7.5